Yosemite/Yosemite/Tools/POS/POSOrderService.swift

```diff
@@ -36,20 +36,21 @@ public extension POSOrder {
     }
 }
 
-struct POSOrderItem {
+public struct POSOrderItem {
     let itemID: Int64
 
     /// The product ID of a product order item, or the ID of the variable product if the order item is a product variation.
-    let productID: Int64
-    let quantity: Decimal
+    public let productID: Int64
+    public let quantity: Decimal
+    public let price: NSDecimalNumber
 
     func toOrderItem() -> OrderItem {
         .init(itemID: itemID,
               name: "",
               productID: productID,
               variationID: .zero,
               quantity: quantity,
-              price: .zero,
+              price: price,
```

Comment:
Since we're changing the implementation of the model, let's unit test as well that it continues to work as expected where this is used.